modules/options/src/main/scala/scala/build/options/BuildOptions.scala

```diff
@@ -58,11 +58,11 @@ final case class BuildOptions(
       case Some(scalaParams0) =>
         val platform0 = platform.value match {
           case Platform.JVM =>
-            val jvmIdSuffix =
+            val jvmVersion =
               javaOptions.jvmIdOpt.map(_.value)
```

Comment:
BTW. do notice that when a particular JVM is passed by a directive or option, its identifier is used directly. Extracting the full Java version out of that will be even trickier.
```bash
scala-cli -e 'println("Hello")' --jvm adoptium:17
# Compiling project (Scala 3.4.2, JVM (adoptium:17))
# Compiled project (Scala 3.4.2, JVM (adoptium:17))
# Hello
```